lib/src/st2110/st_tx_audio_session.c

```diff
@@ -316,6 +316,12 @@ static int tx_audio_session_sync_pacing(struct mtl_main_impl* impl,
   pacing->cur_epoch_time = tx_audio_pacing_time(pacing, epochs);
   pacing->pacing_time_stamp = tx_audio_pacing_time_stamp(pacing, epochs);
   pacing->rtp_time_stamp = pacing->pacing_time_stamp;
+  if (s->ops.rtp_timestamp_delta_us) {
+    double rtp_timestamp_delta_us = s->ops.rtp_timestamp_delta_us;
+    int32_t rtp_timestamp_delta =
+        (rtp_timestamp_delta_us * NS_PER_US) * pacing->pkt_time_sampling / pacing->trs;
+    pacing->rtp_time_stamp += rtp_timestamp_delta;
+  }
```

Comment:
Would removing goto (the if statement)  and changing the above to bellow code by having `s->ops.rtp_timestamp_delta_us=0` by default  equal  0:

```suggestion
    int32_t rtp_timestamp_delta =
        ((double) s->ops.rtp_timestamp_delta_us * NS_PER_US) * pacing->pkt_time_sampling / pacing->trs;
    pacing->rtp_time_stamp += rtp_timestamp_delta;
```
?